ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -55,9 +55,15 @@ fi
 
 
 uorb start
-if [ -f eeprom/parameters ]
```

Comment:
I think that was on purpose here, so that there is no error shown during first startup. Is that still the case?

Reply:
That is right, I fixed it